src/TTrigger.cpp

```diff
@@ -215,16 +216,16 @@ bool TTrigger::setRegexCodeList(QStringList regexList, QList<int> propertyList)
 
             if (!re) {
                 if (mudlet::debugMode) {
-                    TDebug(QColor(Qt::white), QColor(Qt::red)) << "REGEX ERROR: failed to compile, reason:\n" << error << "\n" >> 0;
-                    TDebug(QColor(Qt::red), QColor(Qt::gray)) << R"(in: ")" << regexp.constData() << "\"\n" >> 0;
+                    TDebug(Qt::white, Qt::red) << "REGEX ERROR: failed to compile, reason:\n" << error << "\n" >> mpHost;
+                    TDebug(Qt::red, Qt::white) << TDebug::csmContinue << R"(in: ")" << regexp.constData() << "\"\n" >> mpHost;
```

Comment:
this file contains a bunch of unexpected color changes here, not just removed QColor

Reply:
Another one of the changes I didn't undo. As it happens I found that "red-on-grey" is visually (for me who does have normal colour perception) a poor choice - but will revert for another PR.

Note that all the explicit `QColor(`...`)` wrappers in the `TDebug` constructor uses are redundant and since I was visiting every one of them to put the `Host*` on the end it made sense to remove them at the same time.

Reply:
> will revert for another PR

Good call

> made sense to remove them

Good call

Reply:
You undid this very instance but there were a handful. Please review

Reply:
Done, I think.